x/emissions/module/abci.go

```diff
@@ -73,41 +70,41 @@ func EndBlocker(ctx context.Context, am AppModule) error {
 	workerWindowsToClose := am.keeper.GetWorkerWindowTopicIds(sdkCtx, blockHeight)
 	if len(workerWindowsToClose.TopicIds) > 0 {
 		for _, topicId := range workerWindowsToClose.TopicIds {
-			sdkCtx.Logger().Info(fmt.Sprintf("ABCI EndBlocker: Worker close cadence met for topic: %d", topicId))
+			sdkCtx.Logger().Info("ABCI EndBlocker: Worker close cadence met for topic", "topicId", topicId)
 			// Check if there is an unfulfilled nonce
 			nonces, err := am.keeper.GetUnfulfilledWorkerNonces(sdkCtx, topicId)
 			if err != nil {
-				sdkCtx.Logger().Warn(fmt.Sprintf("Error getting unfulfilled worker nonces: %s", err.Error()))
+				sdkCtx.Logger().Warn("Error getting unfulfilled worker nonces", "error", err.Error())
```

Comment:
```suggestion
				sdkCtx.Logger().Warn("Error getting unfulfilled worker nonces", "error", err)
```

zerolog should manage that

Reply:
ok, this is in many places. Will check for them and put them all together.

Reply:
done